### PR TITLE
Adding 'cinemagoerng' and 'typedload'

### PR DESCRIPTION
**Pull request by HeavyTony2**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by HeavyTony2** [comment]
https://github.com/conda-forge/staged-recipes/pull/28537/commits/a53500d3d48f9116be8506fa9541f4892ffb65b2
Useless file but required but  https://github.com/conda-forge/staged-recipes/issues/28445

**Comment by HeavyTony2** [comment]
@conda-forge/help-python, ready for review

File: recipes/typedload/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+python_min: '3.9'
```
Comment (ocefpaf) [inline recipes/typedload/conda_build_config.yaml:1]: You don't need this file. You only need to define this if the package min is higher than conda-forge's min, which is related to Pyhton's EOL.

File: recipes/cinemagoerng/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+python_min: '3.10'
```
Comment (ocefpaf) [inline recipes/cinemagoerng/conda_build_config.yaml:1]: I prefer to have this defined in the recipe itself, making it easier to remove/update when the time comes.

File: recipes/typedload/conda_build_config.yaml (hunk 1 above)
Comment (HeavyTony2) [reply recipes/typedload/conda_build_config.yaml:1]: I know, but without it, the windows building failed. https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1106791&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de&l=735 `python_min' is undefined `
https://github.com/conda-forge/staged-recipes/issues/28445

Comment (ocefpaf) [reply recipes/typedload/conda_build_config.yaml:1]: It should not. @beckermr do you knwo what is going on?

PS: B/c it is noarch, it is fine to fail in this case. Also, my comment above is about my preference, not a must to merge this.

Comment (HeavyTony2) [reply recipes/typedload/conda_build_config.yaml:1]: @beckermr @ocefpaf Since you're informed of the situation, I'll simply remove it.

**Pull request merged**